ee/dtr/admin/install/index.md

```diff
@@ -10,40 +10,44 @@ Docker Trusted Registry (DTR) is a containerized application that runs on a
 swarm managed by the Universal Control Plane (UCP). It can be installed
 on-premises or on a cloud infrastructure.
 
-## Step 1. Validate the system requirements
+DTR installation is comprised of seven distinct steps:
 
-Before installing DTR, make sure your
-infrastructure meets the [system requirements](./system-requirements) that DTR
-needs to run.
+  1. [Validate System Requirements](validate-the-system-requirements)
+  2. [Install UCP](install-ucp)
+  3. [Install DTR](install-dtr)
+  4. [Confirm that DTR is Running](confirm-that-dtr-is-running)
+  5. [Configure DTR](configure-dtr)
+  6. [Push/Pull Testing](push/pull-testing)
+  7. [Join Replicas to the Cluster](join-replicas-top-the-cluster) (optional)
 
-## Step 2. Install UCP
+## 1. Validate the System Requirements
 
-DTR requires Docker Universal Control Panel (UCP) to run. If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
+Before installing DTR, confirm that your infrastructure meets DTR's [system requirements](./system-requirements).
+
+## 2. Install UCP
+
+To run, DTR requires Docker Enterprise's Docker Universal Control Panel (UCP). If UCP is not yet installed, refer to [install UCP for production](/ee/ucp/admin/install/).
```

Comment:
UCP is already spelled out in the opening sentence.

Reply:
Docker Enterprises Docker (seems like a lot of Docker). 
Not sure what the branding is though.

Reply:
Resolved both comments with one edit.